designer/server/src/main/scala/pl/touk/nussknacker/ui/process/periodic/PeriodicBatchConfig.scala

```diff
@@ -12,17 +12,15 @@ import scala.concurrent.duration._
   * @param rescheduleCheckInterval {@link RescheduleFinishedActor} check interval.
   * @param deployInterval {@link DeploymentActor} check interval.
   * @param deploymentRetry {@link DeploymentRetryConfig}  for deployment failure recovery.
-  * @param jarsDir Directory for jars storage.
   * @param maxFetchedPeriodicScenarioActivities Optional limit of number of latest periodic-related Scenario Activities that are returned by Periodic DM.
   */
 case class PeriodicBatchConfig(
-    db: Config,
+    db: Option[Config],
```

Comment:
Let's keep naming consistent and not introduce another `batch` word. Also let's add a TODO that we want to remove this db parameter after we migrate to the new approach. What about other configuration option? I guess that `processingType` will be also removed?

Reply:
Renamed PeriodicBatchConfig -> SchedulingConfig. Field name `db` changed to `legacyDb` with comment, that it is legacy and will be removed in the future.

Reply:
What about other fields? I guess that `processingType` is also to remove because this config is already inside `processingType`. So why user have to specify it twice?

Reply:
Other fields seem necessary, except processingType. But we cannot remove processing type at this moment. 
- this processing type is an identifier of the DM that stores data in the scheduling DB tables
- it can be different (and in default Nu config is different) than the "true" processingType"
- so it would make the migration much more difficult, I'd rather leave it without changes in this PR

Reply:
> it can be different (and in default Nu config is different) than the "true" processingType"

I don't understand that but let me know if I'm correct or wrong. After migration it can be removed? If so, let's add a comment the same as with legacy db

Reply:
It cannot be removed after migration, because it is a parameter, that identifies the DM, that uses the scheduling mechanism in the db entries. It was a mistake in the past, that it is not the same as the actual, true name of the processingType. All DB queries filter by `processingType`.  

This value can be removed from configuration only after simultaneously modifying all rows in the periodic db + at the same time changing this value in the config to be equal to the true value of processingType.

Reply:
Ok now I'm getting it and this comment is ok - let's clean up this mess eventually.